includes/definitions/omnitron-iconverter.yaml

```diff
@@ -0,0 +1,14 @@
+os: omnitron-iconverter
+text: 'Omnitron iConverter'
+type: management
+icon: omnitron
+mib_dir:
+    - omnitron
+#over:
+    #- { graph: device_current, text: 'Current' }
+    #- { graph: device_voltage, text: 'Voltage' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.7342
+    - sysDescr_regex:
```

Comment:
Please remove sysDescr_regex.
What you have here is:
sysObjectId starts with .1.3.6.1.4.1.7342
OR
sysDescr matches this regex "..."

If you want
sysObjectId starts with .1.3.6.1.4.1.7342
AND
sysDescr contains iConverter
Use this:
```
discovery:
    -
        - sysObjectId: .1.3.6.1.4.1.7342
        - sysDescr: iConverter
```

But it looks like the vendor has properly utilized sysObjectId, so you can just use:
```
discovery:
    - sysObjectId:
        - .1.3.6.1.4.1.7342.5.
```